### DISCUSSION TITLE
How to setup OSD for ardupilot without messing up analog osd?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#377] [Questions]
I'm using SSC338Q+3536 OpenIPC set. 
- UART is properly connected 
- and found that there is mavlink package received by GS unit( from another PC(`wfb-cli gs`) // airunit is transmitting packets

But those icons/digits won't change on 3536, I'm NOT sure what else should I have to change?

Any tips or guides, links is appreciated. 

I tried MSP/Displayport, it seems doesn't work.
```
SERIAL8_PROTOCOL = 2 //MAVLinkv2, on UART8
SERIAL8_BAUD = 115 // default
```

**Comment by svpcom** [comment]
For openipc related questions please ask in openipc-fpv telegram group. I'm not an openipc developer

**Reply by lida2003** [reply]
I saw your video flying ardupilot/betaflight. I didn't find configurations on ardupilot, hope to know the settings on ardupilot, as I did try displayport/msp/mavlink2, all seems not work.

Yeah, I think the best way is ask openipc, but we can't apply telegram account here. But I tryied ardupilot forum https://discuss.ardupilot.org/t/how-to-setup-two-vtx-one-for-analog-camera-another-for-digital-camera/125151

---- EDIT:

Maybe it's something to do with gs configuration: https://github.com/OpenIPC/silicon_research/issues/53

**Comment by lida2003** [comment]
Solution: https://github.com/OpenIPC/silicon_research/issues/53
